components/CollectiveNavbar.js

```diff
@@ -378,11 +380,13 @@ const CollectiveNavbar = ({
   isAnimated,
   createNotification,
   intl,
+  router,
 }) => {
   const [isExpanded, setExpanded] = React.useState(false);
   sections = sections || getFilteredSectionsForCollective(collective, isAdmin);
   callsToAction = { ...getDefaultCallsToActions(collective, isAdmin), ...callsToAction };
   const isEvent = collective?.type === CollectiveType.EVENT;
+  const isPageMainCollectivePage = router.pathname === '/collective-page';
```

Comment:
This seems a bit out of scope from navbar's responsibilities (why should it know about the collective page route?). It also adds some complexity.

As a simpler alterlative, we could just pass a `showBackButton` boolean prop to the component. If you don't want to pass it everywhere, just set its default value to `true`.